generate.py

```diff
@@ -212,7 +213,8 @@ def normalize(code):
     'description': f'Machine translation for { name }',
     'code': code,
     'family': family,
-    'supported_apis': supported_apis
+    'supported_apis': supported_apis,
+    'lan': language,
```

Comment:
I'd call this `'info'`, since it's for the infobox.

Not good to have `language.language` or `lan.lan`.

Reply:
Another option would be to just "unpack" it all into that dict.

```
    'supported_apis': supported_apis,
    **language
```